syncstorage/db.go

```diff
@@ -49,8 +49,9 @@ const (
 	SORT_OLDEST
 	SORT_INDEX
 
-	// Keep BSO for 1 year
-	DEFAULT_BSO_TTL = 365 * 24 * 60 * 60 * 1000
```

Comment:
Does this mean there are bookmarks, passwords, and keys on a production Go server that will magically disappear in 2018?

If so, that's a critical bug that needs to be fixed.